### PR TITLE
feat(docs): add inline documentation to experimental packages

### PR DESCRIPTION
**Pull request by sanderson**
Adds inline documentation to the following packages:

- experimental/http
- experimental/influxdb
- experimental/json
- experimental/mqtt
- experimental/oee

### Done checklist
- [x] docs/SPEC.md updated
- [x] Test cases written

### CODE REVIEW

File: libflux/flux-core/src/bin/fluxdoc.rs [hunk 1]
```diff
@@ -305,11 +305,6 @@ const EXCEPTIONS: &[&str] = &[
     "contrib/sranka",
     "contrib/sranka/teams",
     "contrib/sranka/webexteams",
-    "experimental/http",
-    "experimental/influxdb",
-    "experimental/json",
-    "experimental/mqtt",
-    "experimental/oee",
```
Comment (wolffcm) [inline libflux/flux-core/src/bin/fluxdoc.rs:312]: 🎉

File: stdlib/experimental/influxdb/influxdb.flux [hunk 2]
```diff
@@ -1,8 +1,63 @@
...
+// - **body**: HTTP response body (bytes). 
+// 
+// ## Parameters
+// - method: HTTP request method.
+// - path: InfluxDB API path.
+// - host: InfluxDB host URL _(Required when executed outside of InfluxDB)_.
+//   Default is `""`.
+// - token: [InfluxDB API token](https://docs.influxdata.com/influxdb/cloud/security/tokens/)
+//   _(Required when executed outside of InfluxDB)_.
+//   Default is `""`.
```
Comment (wolffcm) [inline stdlib/experimental/influxdb/influxdb.flux:25]: Is this one not required all the time? I guess we don't require a token for running `to()` for example, so maybe not---just double checking.

File: stdlib/experimental/http/http.flux [hunk 3]
```diff
@@ -1,6 +1,34 @@
...
+// ## Response format
+// `http.get()` returns a record with the following properties:
+// 
+// - **statusCode**: HTTP status code returned by the GET request (int).
+// - **body**: HTTP response body (bytes). 
+// - **headers**: HTTP response headers (record).
+// 
+// ## Parameters
+// - url: URL to send the GET request to.
+// - headers: Headers to include with the GET request.
```
Comment (wolffcm) [inline stdlib/experimental/http/http.flux:21]: Will it be clear to the reader that headers is an object with like `{Authorization: "foo", Connection: "keep-alive"}`? Might be good to include an example. (Similar comment for the other functions that have a parameter like this)

File: stdlib/experimental/influxdb/influxdb.flux [hunk 4]
```diff
@@ -1,8 +1,63 @@
...
+// - headers: HTTP request headers.
```
Comment (wolffcm) [inline stdlib/experimental/influxdb/influxdb.flux:26]: I just noticed that `headers` is a `dict` here and a `record` above... I wish we were consistent.

File: stdlib/experimental/json/json.flux [hunk 5]
```diff
@@ -1,7 +1,52 @@
...
+//                 a: jsonData.a,
+//                 b: jsonData.b,
+//                 c: jsonData.c,
+//             }
+//         },
+// >     )
+// ```
+// 
+// tags: type-conversions
+//
```
Comment (wolffcm) [inline stdlib/experimental/json/json.flux:51]: You can run this one, right?

File: stdlib/experimental/oee/oee.flux [hunk 6]
```diff
@@ -1,17 +1,94 @@
...
-// computeAPQ computes availability, performance, quality and overall equipment effectiveness (oee).
-// productionEvents - a stream of start/stop events for the production process. Each row contains
-//   a _time and state that indicates start and stop events.
-// partEvents - a stream of part counts. Each row contains cumulative counts where column partCount
-//   represents total number of produced parts and badCount number of parts that did not meet quality standards.
-// runningState - production event or state value that corresponds to equipment running state
-// plannedTime - total time that equipment is expected to produce
-// idealCycleTime - theoretical minimum time to produce one part
+// computeAPQ computes availability, performance, and quality (APQ)
+// and overall equipment effectiveness (OEE) using two separate input streams—**production events**
```
Comment (wolffcm) [inline stdlib/experimental/oee/oee.flux:12]: I guess we can't use an em-dash in our docs, I wish we could.

**Review by wolffcm — APPROVED** [review]
This looks great.  I had some comments, but I'll leave any changes up to your discretion. Happy to have another look if you decide to make changes.

File: stdlib/experimental/http/http.flux (hunk 3 above)
Comment (sanderson) [reply stdlib/experimental/http/http.flux:21]: Added headers to the example below.

File: stdlib/experimental/influxdb/influxdb.flux (hunk 2 above)
Comment (sanderson) [reply stdlib/experimental/influxdb/influxdb.flux:25]: So I don't know exactly how this works, but I just tested it in Cloud, and host and token weren't required. 🤷🏻‍♂️ 

I assume it's similar to `to()` where InfluxDB actually uses a different implementation whenever it's executed within the context of InfluxDB.

File: stdlib/experimental/influxdb/influxdb.flux (hunk 4 above)
Comment (sanderson) [reply stdlib/experimental/influxdb/influxdb.flux:26]: Yeah, the `experimental/http` package was added before support for dictionaries. This package was added after. I'm all for consistency as well.

File: stdlib/experimental/json/json.flux (hunk 5 above)
Comment (sanderson) [reply stdlib/experimental/json/json.flux:51]: Yep, nice catch. I'll remove the `no_run` tag.

File: stdlib/experimental/oee/oee.flux (hunk 6 above)
Comment (sanderson) [reply stdlib/experimental/oee/oee.flux]: Replace with a colon.

**Pull request merged**